### ISSUE TITLE
Is it still undetected?

### ISSUE THREAD

**Issue by newbieverse** [JacksonElia/AFK_Apex_Bot#12]
Hi ! just wanted to ask u if your python bot is still undetected as of January 2023

**Comment by LouisMinguet** [comment]
No, you can be ban with this... My friend was banned for cheat by using it only 30min

**Comment by H3rcu1es** [comment]
> No, you can be ban with this... My friend was banned for cheat by using it only 30min

well deserved
